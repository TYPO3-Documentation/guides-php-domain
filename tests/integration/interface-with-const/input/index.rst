========================
PHP Interface with const
========================

..  php:interface:: TYPO3\CMS\Core\TestInterface

    Lorem Ipsum Dolor!

    ..  php:const:: ATOM

            The ATOM
