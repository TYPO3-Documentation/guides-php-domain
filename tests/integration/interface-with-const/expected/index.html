<!-- content start -->
    <div class="section" id="php-interface-with-const">
    <h1>PHP Interface with const</h1>

            <dl class="php interface">
    <dt class="sig sig-object php"
        id="typo3-cms-core-testinterface">
        <em class="property"><span class="pre">interface</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\</span></span>
<span class="sig-name descname"><span class="pre">TestInterface</span></span>
</dt>
    <dd>
        <p>Lorem Ipsum Dolor!</p><dl class="php const">
    <dt class="sig sig-object php" id="typo3-cms-core-testinterface-atom">
        <em class="property"><span class="pre">const</span></em>
        <span class="sig-name descname"><span class="pre">ATOM</span></span>

    </dt>
    <dd><p>The ATOM</p></dd>
</dl>

    </dd>
</dl>

    </div>

<!-- content end -->
